src/Microsoft.Health.Fhir.SqlServer.Api/Controllers/SchemaController.cs

```diff
@@ -72,7 +73,7 @@ public ActionResult SqlScript(int id)
         {
             _logger.LogInformation($"Attempting to get script for schema version: {id}");
 
-            throw new NotImplementedException(Resources.ScriptNotImplemented);
+            return Content(SchemaUpgradeRunner.GetMigrationScript(id));
```

Comment:
We need to specify the content type here of `application/sql`. Additionally, we probably want to change the return type `File` and also specify a filename which will then set the `Content-Disposition` header.

Reply:
Addressed it 👍